v1.0.0.md

```diff
@@ -31,8 +31,8 @@ Harvester can be installed from
 * Enhanced Harvester cluster import experience to Rancher. [[DOC](https://docs.harvesterhci.io/v1.0/rancher/virtualization-management/)]
 * Support enable/disable additional data storage partition. [[DOC](https://docs.harvesterhci.io/v1.0/install/harvester-configuration/#installno_data_partition)]
 * Support volume hot-unplugin. [[DOC](https://docs.harvesterhci.io/v1.0/vm/hotplug-volume/)]
-* Multi-disk support enhancement. [[LINKS](https://github.com/harvester/harvester/issues?q=is%3Aissue+label%3Aarea%2Fnode-disk-manager+milestone%3Av1.0.0+)]
 * Support resource over-commit config. [[DOC](https://docs.harvesterhci.io/v1.0/settings/settings/#overcommit-config)]
+* (Experimental) Multi-disk support enhancement. [[LINKS](https://github.com/harvester/harvester/issues?q=is%3Aissue+label%3Aarea%2Fnode-disk-manager+milestone%3Av1.0.0+)]
```

Comment:
Not multi-disk support. Strictly to automatic provisioning.

Reply:
We can call it out for that feature and point to https://docs.harvesterhci.io/v1.0/settings/settings/#auto-disk-provision-paths

Reply:
sure, then will a new line of it, thanks.